FashionConnect_ui/src/components/Collections/Collections.jsx

```diff
@@ -24,13 +41,29 @@ export default function Collections({ collections, products, setCollections }) {
                         : collection
                 ));
             } else {
-                console.error('Failed to delete product from the collection');
+                throw error('Failed to delete product from the collection');
             }
         } catch (error) {
-            console.error('Error deleting product:', error);
+            throw error
```

Comment:
Out of curiosity, was this pattern suggested somewhere?

Reply:
Yeah, Syed said that it is not ideal to push console messages to production.
However, i use them to debug but i delete them before pushing the files.